.gitignore

```diff
@@ -6,3 +6,4 @@ access_logs.db
 npm-debug.log
 dist
 db/db.json
+.idea
```

Comment:
@DanilAgafonov We shouldn't list such specific entries in project's `.gitignore`. Imagine a situation, when each developer uses its own editor - this will quickly become a mess. If you're using JetBrains tools, just ignore the `.idea` folder in your local `.gitignore` (`~/.gitignore`)